avaframe/data/avaSimilaritySol/Inputs/simiSol_com1DFACfg.ini

```diff
@@ -16,14 +16,17 @@ rhoEnt = 200
 massPerPart = 500
 # minimum flow depth [m]
 hmin = 0.05
+#+++++++++++++SPH parameters
+# SPH gradient option
+sphOption = 1
```

Comment:
here it is 1 and in the one above for damBreak it is 4, why not use the same one? Could you indicate here also that this is a TODO to update once we have a 'final' one?